src/XMakeBuildEngine/UnitTests/Evaluation/ImportFromMSBuildExtensionsPath_Tests.cs

```diff
@@ -349,14 +347,6 @@ public void ImportFromExtensionsPathAnd32And64()
                 </Project>
                 ";
 
-            string v2Folder = NativeMethodsShared.IsWindows
```

Comment:
Note: These 2 variables were defined but not used so I just deleted them (since IsWindows doesn't exist in master).